src/main/perl/RuleBasedEditor.pm

```diff
@@ -437,10 +454,31 @@ sub _formatAttributeValue
         }
         $formatted_value = join " ", @$attr_value;
 
+    } elsif ( $value_fmt == LINE_VALUE_HASH && !($value_opt & LINE_VALUE_OPT_SINGLE) ) {
+        $self->debug(2, "$function_name: hash received with keys: ", join(",",(sort keys %$attr_value)));
+        # Key/value separator (D: space)
+        my $key_val_separator = ' ';
+        $key_val_separator = ':' if ($value_opt & LINE_VALUE_OPT_SEP_COLON);
+        # Prefix to add before key (D: none)
+        my $key_prefix = '';
+        $key_prefix = '-' if ($value_opt & LINE_KEY_OPT_PREFIX_DASH);
+        $self->debug(2,"$function_name: key prefix: >>>$key_prefix<<<; key/value separator: $key_val_separator");
+        my @tmp_values;
+        foreach my $k (sort keys %$attr_value) {
+            my $v = $attr_value->{$k};
+            # Keys may be escaped if they contain characters like '/': unescaping a non-escaped
+            # string is generally harmless.
+            my $tmp = $key_prefix.unescape($k).$key_val_separator.$v;
+            $self->debug(2,"$function_name: hash key/value string: '".$tmp);
```

Comment:
space after comma, there probably needs to be a `."'" after the `$tmp` (but you use `$tmp` directly in the double quoted string)